recipes/dart/build.sh

```diff
@@ -0,0 +1,3 @@
+cp -R bin $PREFIX
```

Comment:
Please add `set -e`

Reply:
Also is there any way to actually build dart, rather than relying on pre-built binaries?

Reply:
Yes. It is possible to build the Dart SDK from source, but the pre-built version probably costs me less time and uses less conda-forge resources. Additionally I don't have a macOS system to test on. You're free to insist that this Dart SDK is built using the conda-forge compilers, however.

Reply:
Yep the default is to build on conda-forge.  We only repackage binaries in exceptional circumstances

Reply:
No need to worry about CI time resources

Reply:
Aight. I'll be back in a month with a recipe that builds dart from source.